data/korath/korath weapons.txt

```diff
@@ -577,6 +737,138 @@ effect "slicer impact"
+		sound "korath digger"
+		"hit effect" "korath digger hit"
+		"inaccuracy" 1.5
+		"turret turn" 4.6
+		"velocity" 480
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 2.9
+		"firing heat" 4.9
+		"shield damage" 0.8
+		"hull damage" 1.3
+		"minable damage" 5.2
+		"disruption damage" .08
+	description `When the Korath Exiles needed a cheap mining laser for their large vessels, they re-engineered this ancient but robust design. The mild shield disruption effect makes it somewhat useful in combat.`
+
+outfit "Korath Digger"
+	category "Guns"
+	cost 419000
+	thumbnail "outfit/korath digger"
+	"mass" 10
+	"outfit space" -10
+	"weapon capacity" -10
+	"asteroid scan power" 10
+	"gun ports" -1
+	weapon
+		sprite "projectile/digger thin"
+			"frame rate" 20
+		sound "korath digger"
+		"hit effect" "korath digger hit"
+		"inaccuracy" 0.1
+		"velocity" 480
+		"lifetime" 1
+		"reload" 1
+		"firing energy"  2.1
+		"firing heat" 4.3
+		"shield damage" 0.5
+		"hull damage" 0.9
+		"minable damage" 4.6
+		"disruption damage" .06
+	description `The gun version of the classic Korath mining laser allows a larger mothership to release a cloud of fighter-sized miners to harvest resources when working faster-moving or more dangerous asteroid fields.`
+
+effect "korath digger hit"
+	sprite "effect/korath digger hit"
+		"no repeat"
+		"frame rate" 30
+	"lifetime" 5
+	"random angle" 30
+	"velocity scale" .005
+
+
+outfit "Korath Heaver"
+	category "Secondary Weapons"
+	cost 3450000
+	thumbnail "outfit/korath heaver"
+	"mass" 46
+	"outfit space" -46
+	"weapon capacity" -46
+	"fuel capacity" 200
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/heaver hyperspace"
+			"frame rate" 12
+			rewind
+		"hardpoint sprite" "hardpoint/korath heaver"
+		icon "icon/korath heaver"
+		velocity 30
+		lifetime 8
+		"hit force" 3600
+		"hit effect" "heaver hyperspace hit"
+		sound "heaver"
+		submunition "Heaver Shard" 20
+		"turret turn" 2.2
+		"reload" 40
+		"burst reload" 12
+		"burst count" 6
+		"firing energy" 31
+		"firing heat" 190
+		"firing fuel" 12.5
+		"inaccuracy" 0.5
+	description `Korath Heavers raise to a fine art the age-old warfare method of throwing rocks at one's enemy until they die. A small Banisher throws asteroids or other debris into an unstable hyperspace window. At close range, this merely pushes the enemy, but after the debris exits hyperspace, it becomes a rapidly-expanding cloud of deadly projectiles.`
+
+effect "heaver hyperspace hit"
+	sprite "effect/heaver hyperspace hit"
+		"no repeat"
+		rewind
+		"frame rate" 60
+	lifetime 36
+	"velocity scale" .1
+	sound "heaver hyperspace hit"
+
+outfit "Heaver Shard"
+	weapon
+		sprite "projectile/heaver shard"
+			"frame rate" 12
+			"rewind"
+		"random velocity" 3
+		"velocity" 10
+		"lifetime" 40
+		"random lifetime" 15
+		"shield damage" 17
+		"hull damage" 17
+		"inaccuracy" 3.5
+		"hit force" 20
+		"hit effect" "heaver shard hit" 5
+		"missile strength" 12
```

Comment:
I don't want to shoot down suggestions just because this is a late stage.  However, going into finer and finer projectiles, while interesting, may be a bit unnecessary and simulation-y.  In practise they do spread out and fade, and that's enough to suggest the effect to me.